src/site/content/en/blog/cors-rfc1918-feedback/index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: \[Feedback wanted\] CORS for private networks (RFC1918)
```

Comment:
```suggestion
title: Feedback wanted: CORS for private networks (RFC1918)
```

Reply:
This causes an issue: https://github.com/GoogleChrome/web.dev/issues/4194

Reply:
Removing `[` and `[` sounds fair enough

Reply:
Looks like `:` is causing Netlify to build at this time. Changing the title...